sample.env

```diff
@@ -241,30 +228,31 @@ OBU_OTA_LOGGING_LEVEL="INFO"
 # Supported log levels are "ALL", "DEBUG", "ERROR", "FATAL", "INFO", "OFF", "TRACE" and "WARN"
 KC_LOGGING_LEVEL="INFO"
 
-# Set the MongoDB dataset to load, specified in the conflictmonitor/datasets directory
-CONFLICTMONITOR_MONGODB_INITIAL_DATASET="uturn_export"
-
 ##########################################
 # JPO ConflictMonitor Docker Environment #
 # Documentation: https://github.com/usdot-jpo-ode/jpo-conflictmonitor
 ##########################################
-
-# (Required) The IP address of Docker host machine which can be found by running "ifconfig"
-# Hint: look for "inet addr:" within "eth0" or "en0" for OSX
-# For Local deployments DB_HOST_IP and KAFKA_BROKER_IP should be set to DOCKER_HOST_IP, for remote deployments, these can be set independently.
 DB_HOST_IP=${DOCKER_HOST_IP}
 KAFKA_BROKER_IP=${DOCKER_HOST_IP}
-KAFKA_CONNECT_IP=localhost
 
 # The Username and passwords to use for accessing mongoDB.
 MONGO_INITDB_ROOT_USERNAME=root
-MONGO_INITDB_ROOT_PASSWORD=root
+MONGO_INITDB_ROOT_PASSWORD=
 CM_MONGO_CONNECTOR_USERNAME=connector
-CM_MONGO_CONNECTOR_PASSWORD=connector
+CM_MONGO_CONNECTOR_PASSWORD=
 CM_MONGO_API_USERNAME=api
-CM_MONGO_API_PASSWORD=api
+CM_MONGO_API_PASSWORD=
 CM_MONGO_USER_USERNAME=user
-CM_MONGO_USER_PASSWORD=user
+CM_MONGO_USER_PASSWORD=
+
+CM_DATABASE_NAME=ConflictMonitor
+CM_DATABASE_STORAGE_COLLECTION_NAME=MongoStorage
+CM_DATABASE_SIZE_GB=1000

```

Comment:
Should we reduce this db size for debugging purposes? 1TB is pretty huge for development machines

Reply:
reduced to 50 gb